packages/cli/codemods/add-plugin.js

```diff
@@ -0,0 +1,70 @@
+module.exports = function addPlugin(fileInfo, api, options = {}) {
+    const j = api.jscodeshift
+    const ast = j(fileInfo.source)
+
+    const { pluginInits } = options
+
+    if (!pluginInits) {
+        console.error(
+            new Error(
+                'You need to provide an array of plugins to be initialized'
+            )
+        )
+
+        return
+    }
+
+    if (!Array.isArray(pluginInits)) {
+        console.error(new Error('"pluginInits" must be an array'))
+        return
+    }
+
+    const existingPluginInits = ast
+        .find(j.CallExpression)
+        .filter(path => pluginInits.includes(path.value.callee.name))
+        .nodes()
+        .map(path => path.callee.name)
+
+    const missingPluginInits = pluginInits.filter(
+        pluginInit => !existingPluginInits.includes(pluginInit)
+    )
+
+    const callBody = ast
+        .find(j.BlockStatement)
+        .filter(
+            path =>
+                path.parentPath &&
+                path.parentPath.parentPath &&
+                path.parentPath.parentPath.value &&
+                path.parentPath.parentPath.value.left &&
+                path.parentPath.parentPath.value.left.object &&
+                path.parentPath.parentPath.value.left.property &&
+                path.parentPath.parentPath.value.left.object.name ===
+                    'module' &&
+                path.parentPath.parentPath.value.left.property.name ===
+                    'exports'
```

Comment:
Man, optional chaining (Node >=14) would be amazing here... :cry: 

 ```js
path.parentPath?.value?.left?.property?.name === 'module' && path.parentPath?.value?.left?.object?.name === 'exports'
```

Reply:
yeah...